app/controllers/jobseekers/passwords_controller.rb

```diff
@@ -14,9 +14,4 @@ def ensure_reset_password_period_valid
   def after_sending_reset_password_instructions_path_for(_resource)
     jobseekers_check_your_email_password_path
   end
-
-  def after_resetting_password_path_for(resource)
```

Comment:
This is needed to sign in the and redirect the user to `jobseekers_saved_jobs_path`

Reply:
See the commit message! It's not needed.

Reply:
Maybe when you were writing this controller, the routes.rb did not yet contain 'jobseeker_root', which is what Devise uses to determine the path for after sign in and for after resetting the password.

Reply:
We need to sign in though still, don't we?